app/forms/openid_connect_token_form.rb

```diff
@@ -96,7 +96,6 @@ def validate_client_assertion
                                    sub: client_id, verify_sub: true)
     validate_aud_claim(payload)
   rescue JWT::DecodeError => err
-    # TODO: i18n these JWT gem error messages
```

Comment:
These are not user facing, where user = site visitor. I'm not sure it's worth localizing them. We don't localize other API error messages.